package.json

```diff
@@ -96,6 +96,7 @@
         "@babel/preset-typescript": "^7.13.0",
         "@babel/types": "^7.14.0",
         "@napi-rs/cli": "^2.10.0",
+        "@swc/core": "^1.2.218",
```

Comment:
Will this work with npm publish?

Reply:
I haven't able to actually publish, but I guess it may ok since this is devdependencies. Unfortunately I don't have good way to verify it other than `publish --dry-run` (which seems to pass)

Reply:
Then I think it's fine. I didn't know node js supports cycle in dev deps, though.